### PR TITLE
add missing img alt text

### PR DESCRIPTION
**Pull request by kirsty-hames**
resolves https://github.com/adaptlearning/adapt_framework/issues/3200

- remove aria-label from `` as parent `aria-hidden` prevents label from being read
- apply graphic alt text to `<input>` `aria-label` instead

### CODE REVIEW

File: js/adapt-contrib-gmcq.js [hunk 1]
```diff
@@ -49,6 +49,51 @@ define([
...
       this.$('.js-mcq-item').css('width', isLarge ? (100 / columns) + '%' : '');
+    },
+
+    updateMarking: function () {
+
+      var isInteractive = this.model.isInteractive();
+      var canShowMarking = this.model.get('_canShowMarking');
+      var ariaLabels = Adapt.course.get('_globals')._accessibility._ariaLabels;
+
+      this.model.getChildren().each(function (itemModel) {
```
Comment (tomgreenfield) [inline js/adapt-contrib-gmcq.js:60]: Prefer native alias:
```suggestion
      this.model.getChildren().forEach(function(itemModel) {
```

File: js/adapt-contrib-gmcq.js [hunk 2]
```diff
@@ -49,6 +49,51 @@ define([
...
+          (shouldBeSelected ? ariaLabels.correct : ariaLabels.incorrect),
+          ', ',
+          (isActive ? ariaLabels.selectedAnswer : ariaLabels.unselectedAnswer),
+          '. ',
+          $.a11y_normalize(itemModel.get('text')),
+          '. ',
+          $.a11y_normalize(itemModel.get('_graphic').alt)
+        ].join(''));
+
+      }.bind(this));
```
Comment (tomgreenfield) [inline js/adapt-contrib-gmcq.js:96]: Second argument to `.forEach()` is the context:
```suggestion
      }, this);
```

File: js/adapt-contrib-gmcq.js [hunk 3]
```diff
@@ -49,6 +49,51 @@ define([
 
       this.$el.toggleClass('has-column-layout', isLarge);
       this.$('.js-mcq-item').css('width', isLarge ? (100 / columns) + '%' : '');
+    },
+
+    updateMarking: function () {
```
Comment (tomgreenfield) [inline js/adapt-contrib-gmcq.js:54]: Spacing:
```suggestion
    updateMarking: function() {
```

File: js/adapt-contrib-gmcq.js [hunk 4]
```diff
@@ -49,6 +49,51 @@ define([
...
+        var $itemInput = this.$('.js-item-input').filter('[data-adapt-index="' + index + '"]');
+        var $item = $itemInput.parents('.js-mcq-item');
+
+        if (isInteractive || !canShowMarking) {
+          // Remove item marking
+          $item.removeClass('is-correct is-incorrect');
+          $itemInput.attr('aria-label', [
+            $.a11y_normalize(itemModel.get('text')),
+            '. ',
+            $.a11y_normalize(itemModel.get('_graphic').alt)
```
Comment (tomgreenfield) [inline js/adapt-contrib-gmcq.js:72]: There is a new API for this:
```suggestion
            Adapt.a11y.normalize(itemModel.get('text')),
            '. ',
            Adapt.a11y.normalize(itemModel.get('_graphic').alt)
```
See also lines 91 & 93.

@oliverfoster, probably best to leave the switch to the direct `a11y.normalize()` for the React PR.

Comment (oliverfoster) [reply js/adapt-contrib-gmcq.js:72]: Nah, do it now.

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Comment by oliverfoster** [comment]
@kirsty-hames could you have a look at this this week if you have time please?

**Review by tomgreenfield — APPROVED** [review]
:eye:

**Review by oliverfoster — APPROVED** [review]

File: js/adapt-contrib-gmcq.js (hunk 4 above)
Comment (tomgreenfield) [reply js/adapt-contrib-gmcq.js]: > See also lines 91 & 93.

Future note: this bit wasn't done.

Comment (oliverfoster) [reply js/adapt-contrib-gmcq.js]: Apologies, my mistake.

**Pull request merged**